lib/controllers/login.js

```diff
@@ -33,10 +33,24 @@ module.exports = function(req, res) {
       options = {};
     }
 
+    var hasSocialProviders = false;
+
+    // Iterate all social providers and check if there are any enabled.
+    if (config.socialProviders) {
+      for (var name in config.socialProviders) {
+        var provider = config.socialProviders[name];
+        if (provider.enabled) {
+          hasSocialProviders = true;
+          break;
+        }
+      }
+    }
+
```

Comment:
Nope, [`some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) is a native method on the `Array` prototype and you don't need either underscore or lodash for that. Besides, what I can see none of them are included in this project and including them just for this would be overkill in my opinion.